src/Main/Main.js

```diff
@@ -1,14 +1,19 @@
 import React from 'react';
 import backgroundImg from '../background.png';
 import './Main.css';
+//import '../ZodiacCard/ZodiacCard.css';
+import ZodiacCard from '../ZodiacCard/ZodiacCard.js';
 
-export default function Main() {
+export default function Main(props) {
```

Comment:
This looks great! If you want to you can also destructure the props like `Main({ array })` and then on line 10 you only need to pass in `array.map` instead of `props.array.map`.